rootston/seat.c

```diff
@@ -590,6 +590,15 @@ void roots_seat_focus_view(struct roots_seat *seat, struct roots_view *view) {
 	wlr_seat_keyboard_notify_enter(seat->seat, view->wlr_surface);
 }
 
+void roots_seat_cycle_focus(struct roots_seat *seat) {
```

Comment:
I think it would be more usable to focus the previously focused view here, which is the second in the list. Take the first view in the list and put it at the back to preserve the order.

Reply:
Yeah. Done!